### DISCUSSION TITLE
Lesson 6 - Ganache Installer not working on my Windows system

### DISCUSSION THREAD

**Discussion by dezlymacauley** [Cyfrin/foundry-full-course-cu#381] [Q&A] [answered]
As soon as I open the installer that I downloaded from https://trufflesuite.com/ganache/
I get the error:

"Cannot open app package"
"Reason: Error in parsing the app package."

**Comment by dezlymacauley** [comment, marked as answer]
Update: Got it working. 

For anyone else experiencing this error, here's what I did.
1. Open up the Windows terminal (I used PowerShell) in administrator mode
2. cd to the folder where your Ganache-2.7.1-win-x64 file is
```powershell
cd "C:\Users\User\Downloads"
````
Put the file path in quotes in case there are spaces in the folder names

3. Run the following command:
```powershell
Add-AppxPackage -Path .\Ganache-2.7.1-win-x64.appx
```

**Comment by adesokan247** [comment]
i couldn't even download, i dont know if you can help me with this error

i get file wasn't available on site as error while downloading
